ruddock/modules/admin/member_helpers.py

```diff
@@ -10,14 +10,14 @@ class NewMember:
   """Class containing data for adding a single new member."""
   def __init__(self, first_name, last_name, matriculation_year, graduation_year,
       uid, email, membership_desc):
-    self.first_name = first_name
-    self.last_name = last_name
-    self.name = first_name + ' ' + last_name
-    self.matriculation_year = matriculation_year
-    self.graduation_year = graduation_year
-    self.uid = uid
-    self.email = email
-    self.membership_desc = membership_desc
+    self.first_name = first_name.encode('utf-8')
+    self.last_name = last_name.encode('utf-8')
+    self.name = (first_name + ' ' + last_name).encode('utf-8')
+    self.matriculation_year = matriculation_year.encode('utf-8')
```

Comment:
Do the the matriculation year/graduation year/UID really need to be encoded? They should probably be numbers and definitely be ASCII.

Reply:
You're totally right, I just kinda carelessly encoded everything because the whole string that `__str__` returns needs to be encoded, but I made that much cleaner now (only the names are encoded, and `__str__` decodes them and encodes the whole string after compiling it). Thanks!